cicd/gitops/configuring-sso-for-argo-cd-using-keycloak.adoc

```diff
@@ -0,0 +1,26 @@
+[id="configuring-sso-for-argo-cd-using-keycloak"]
+= Configuring SSO for Argo CD using keycloak
```

Comment:
Capitalize Keycloak in the title since you capitalize it throughout the modules.